apps/errors/errors.go

```diff
@@ -75,3 +76,19 @@ func (e CallErr) Verbose() string {
 	e.Resp.TLS = nil     // Same
 	return fmt.Sprintf("%s:\nRequest:\n%s\nResponse:\n%s", e.Err, prettyConf.Sprint(e.Req), prettyConf.Sprint(e.Resp))
 }
+
+// Unwrap returns the wrapped error
+func (e CallErr) Unwrap() error {
```

Comment:
I think we should remove this for now, we can always add it later if needed (removing is much harder).  Having an Unwrap() means we could be leaking the inner error as part of our method contract.

See https://blog.golang.org/go1.13-errors, section "Whether to Wrap".

Reply:
Removed in this commit https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/178/commits/5fbd566d1d01afddf610abb63fa29e05fff314ce